cmd/minikube/cmd/start.go

```diff
@@ -162,6 +164,7 @@ func initMinikubeFlags() {
 	startCmd.Flags().String(containerRuntime, "docker", "The container runtime to be used (docker, crio, containerd).")
 	startCmd.Flags().Bool(createMount, false, "This will start the mount daemon and automatically mount files into minikube.")
 	startCmd.Flags().String(mountString, constants.DefaultMountDir+":/minikube-host", "The argument to pass the minikube mount command on start.")
+	startCmd.Flags().StringArrayVar(&addonList, addons, nil, "Enable addons. see addon list if you want to check them `minikube addons list`")
```

Comment:
remove repetitive addon list.
also please provide before after output in the PR description.

Reply:
@medyagh Thank you for review!
I removed repetitive addon list.

Before
```
"Enable addons. see addon list if you want to check them `minikube addons list`"
```

After
```
"Enable addons. see `minikube addons list` if you want to check"
```

And I edited the PR description(user-facing change and Additional documentation). 
Should I also include this above change in the PR description?

If I misunderstand your intention, please let me know.